tests/e2e/improved_csi_idempotency.go

```diff
@@ -625,13 +697,41 @@ func createVolumeWithServiceDown(serviceName string, namespace string, client cl
 					"kubernetes", volumeID))
 		}
 	}()
+
+	if !vanillaCluster {
+		totalquota_used_after, _ = getTotalQuotaConsumedByStoragePolicy(ctx, restConfig, storageclass.Name, namespace)
+		framework.Logf("totalquota_used_after :%v", totalquota_used_after)
+
+		storagepolicyquota_pvc_after, _ = getStoragePolicyQuotaForSpecificResourceType(ctx, restConfig,
+			storageclass.Name, namespace, volExtensionName)
+		framework.Logf("storagepolicyquota_pvc_after :%v", storagepolicyquota_pvc_after)
+
+		storagepolicy_usage_pvc_after, _ = getStoragePolicyUsageForSpecificResourceType(ctx, restConfig,
```

Comment:
Please use camelCase to name your variables

Reply:
updated